manim/scene/vector_space_scene.py

```diff
@@ -444,7 +444,7 @@ def vector_to_coords(self, vector, integer_labels=True, clean_up=True):
         y_line = Line(x_line.get_end(), arrow.get_end())
         x_line.set_color(X_COLOR)
         y_line.set_color(Y_COLOR)
-        x_coord, y_coord = array.get_mob_matrix().flatten()
+        x_coord, y_coord = array.elements
```

Comment:
```suggestion
        x_coord, y_coord = array.get_entries()
```
I think it would be better like this as it might be easier to understand. It does more or less the same thing you suggested.

Reply:
Yeah that sounds good! Thanks for the prompt response :)